src/olympia/abuse/utils.py

```diff
@@ -245,8 +250,19 @@ class CinderActionEscalateAddon(CinderAction):
     def process_action(self):
         """This will return always return a falsey value because we've not taken any
         action at this point, just flagging for human review."""
+        self.flag_for_human_review()
+
+    def flag_for_human_review(self):
```

Comment:
... not sure what the optimization is here.

Reply:
Not really an optimization, just thought it'd be clearer that way